### PR TITLE
SignalR json saving

### PR DESCRIPTION
**Pull request by Sc0rpie**
What has been added:

- Real-time chat is now being saved and loaded upon webpage load.
- For the time being, chat history is being saved in JSON.
- Messages now appear with a timestamp (shows the time when the message was sent).

### CODE REVIEW

**Review by HubertasVin — DISMISSED** [review]
Ooo, reikia čia biški pratrinti

File: Project/InOut/JSONParser.cs
Comment (aWhiteScreen) [inline Project/InOut/JSONParser.cs:1]: we should consider moving the code from the JSONParser constructor to elsewhere, it's probably bad practice.

**Review by aWhiteScreen — CHANGES_REQUESTED** [review]
Project/InOut/JSONParser.cs should probably be changed a bit. Weird comments and unnecessary in the constructor.

**Review by aWhiteScreen — APPROVED** [review]
Looke good now. Approved.

**Review by Sc0rpie — COMMENTED** [review]
Unnecessary code has been removed.

**Pull request merged**